distribution/tools/plugin-cli/src/main/java/org/elasticsearch/plugins/RemovePluginCommand.java

```diff
@@ -84,6 +84,12 @@ void execute(Terminal terminal, Environment env, String pluginName, boolean purg
             throw new UserException(ExitCodes.USAGE, "plugin name is required");
         }
 
+        if ("ingest-geoip".equals(pluginName)) {
```

Comment:
Don't they still need to be able to remove? What if they upgrade before removing the old plugin, then without the remove command working it would have to be done manually to avoid errors later?

Reply:
Good call @rjernst. I pushed a commit to address this. Let me know what you think.